### ISSUE TITLE
Still MODULE_NOT_FOUND after successful push via GH actions

### ISSUE THREAD

**Issue by Nettitude-MS** [microsoft/azurechatgpt#78]
I'm still seeing issues and I am really not sure why as I am confident I have setup the application based on the README.

I've created the required App Reg within my Development AAD and I created the AZURE_CREDENTIALS and the AZURE_APP_SERVICE variables within GH.

I ensured the App Reg had the required Contributor role within the RG I was deployed into.

Used the updated ARM template to ensure that the NEXTAUTH_URL had the 'https:/' prefix as explained previously.

Updated the settings within the App Service with the AZURE_AD_TENANT_ID, AZURE_AD_CLIENT_ID and AZURE_AD_CLIENT_SECRET variables.

Pushed the app files to the app service via the GH action and this excutes successfully without issue.

I then traverse to the app, complete initial sign in (which completes successfully based on the AAD logs)until I get the following error:

![image](https://github.com/microsoft/azurechatgpt/assets/100838612/8716993c-4c25-45d0-b2bd-6a0ee9eb8f36)

I know from previous issues that this related to an issue with the COSMOSDB_KEY and COSMOSDB_URI variables being incorrectly set so I have validated this on my side and I can confirm that the variable names are correct in the App Service and in the code within GH.

I took a look at the Kudu portal to see if I could diagnose further and I am still seeing events like the below:

![image](https://github.com/microsoft/azurechatgpt/assets/100838612/c525b477-8f38-4213-88e5-f84c60a95e02)

I checked the file path that the error mentions (/home/site/wwwroot/) and confirmed that the file 'server.js" does indeed exist within the application files:

![image](https://github.com/microsoft/azurechatgpt/assets/100838612/ade8918d-2cc6-46eb-88f2-afad46760a0e)

Really sorry if this is a simple fix but I am really struggling and not sure where else to troubleshoot. 

Thanks for your help.

**Comment by Nettitude-MS** [comment]
Yes, I have the start-up command entered correctly as far as I can see within General Settings.

My enviorment variables within the Azure Portal are below for:

![image](https://github.com/microsoft/azurechatgpt/assets/100838612/8af4470c-f2be-4f3f-b877-5486af90d104)

My GH secrets are below:

![image](https://github.com/microsoft/azurechatgpt/assets/100838612/6720a839-1c9a-4568-9fe5-c7ec1042bb89)

Anything else I can check?

**Comment by TimArrow** [comment]
Hi there, just deployed the app to an appservice and run into this issue as well. Any idea for a fix?

I do have node server.js set as startup command in my Azure Web App
Variables look nearly like Nettitude-MS ones

2023-08-10T09:57:31 Welcome, you are now connected to log-streaming service.
Starting Log Tail -n 10 of existing logs ----
/home/LogFiles/__lastCheckTime.txt (https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/__lastCheckTime.txt)08/10/2023 09:43:14
/home/LogFiles/kudu/trace/18a4785f3f88-[redacted]-27b1-48f6-9e13-a563904832fc.txt ([https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/kudu/trace/18a4785f3f88-[redacted]-27b1-48f6-9e13-a563904832fc.txt](https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/kudu/trace/18a4785f3f88-%5Bredacted%5D-27b1-48f6-9e13-a563904832fc.txt))
2023-08-10T09:38:04 Startup Request, url: /api/settings, method: GET, type: request, pid: 92,1,29, ScmType: None
/home/LogFiles/kudu/trace/40c8cf713838-[redacted]-ffb7-4102-9a24-736ac6b998a4.txt ([https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/kudu/trace/40c8cf713838-[redacted]-ffb7-4102-9a24-736ac6b998a4.txt](https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/kudu/trace/40c8cf713838-%5Bredacted%5D-ffb7-4102-9a24-736ac6b998a4.txt))
2023-08-10T09:52:35 Startup Request, url: /api/logstream/, method: GET, type: request, pid: 92,1,18, ScmType: None
/home/LogFiles/kudu/trace/722388e64825-[redacted]-936d-45ad-9f65-08d1f1c676c8.txt ([https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/kudu/trace/722388e64825-[redacted]-936d-45ad-9f65-08d1f1c676c8.txt](https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/kudu/trace/722388e64825-%5Bredacted%5D-936d-45ad-9f65-08d1f1c676c8.txt))
2023-08-10T09:26:20 Startup Request, url: /detectors, method: GET, type: request, pid: 84,1,7, ScmType: None
/home/LogFiles/2023_08_10_ln1xsdlwk000006_default_docker.log (https://timsai-[redacted].scm.azurewebsites.net/api/vfs/LogFiles/2023_08_10_ln1xsdlwk000006_default_docker.log)
2023-08-10T09:37:45.890815395Z at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
2023-08-10T09:37:45.890819196Z at Module._load (node:internal/modules/cjs/loader:920:27)
2023-08-10T09:37:45.890823496Z at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
2023-08-10T09:37:45.890840596Z at node:internal/main/run_main_module:23:47 {
2023-08-10T09:37:45.890845096Z code: 'MODULE_NOT_FOUND',
2023-08-10T09:37:45.890849197Z requireStack: []
2023-08-10T09:37:45.890853297Z }
2023-08-10T09:37:45.890857197Z
2023-08-10T09:37:45.890861497Z Node.js v18.16.0

![image](https://github.com/microsoft/azurechatgpt/assets/141012494/0f90712c-65f3-47ba-a327-33c5b8b2b826)

**Comment by Nettitude-MS** [comment]
This must be authentication issue as I have invited a guest account from a separate tenancy and I was able to sign in and use the application without issue.

Must be some kind of bug. Hopefully an admin can help us understand what next steps.
